erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -126,6 +126,12 @@ frappe.ui.form.on("BOM", {
 		}
 	},
 
+	item: function(frm) {
+		frappe.db.get_value('Item', {name: frm.doc.item}, 'inspection_required_before_manufacturing', (r) => {
```

Comment:
Add a condition to only call this function if an item is selected, and not when empty